core/src/main/scala/org/apache/spark/sql/execution/ui/SnappyStatsPage.scala

```diff
@@ -0,0 +1,82 @@
+/*
+ * Changes for SnappyData additions and modifications.
+ *
+ * Portions Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
```

Comment:
@nthanvi If some code for this has been originally taken from Apache Spark's WebUI or similar, then attach the Apache ASL license before this, else change this to that for new files like in SnappyContext.scala